third_party/bigframes_vendored/pandas/core/series.py

```diff
@@ -2711,6 +2711,31 @@ def argmax(self):
 
         If the minimum is achieved in multiple locations, the first row position is returned.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+        Consider dataset containing cereal calories.
+
+            >>> s = bpd.Series({'Corn Flakes': 100.0, 'Almond Delight': 110.0,
+            ...                 'Cinnamon Toast Crunch': 120.0, 'Cocoa Puff': 110.0})
+            >>> s
+            Corn Flakes              100.0
+            Almond Delight           110.0
+            Cinnamon Toast Crunch    120.0
+            Cocoa Puff               110.0
+            dtype: Float64
+
+            >>> s.argmax()
```

Comment:
argmin and argmax both present for argmax, same in the argmin code sample

Reply:
I'm referring to pandas page: 
https://pandas.pydata.org/docs/reference/api/pandas.Series.argmax.html and https://pandas.pydata.org/docs/reference/api/pandas.Series.argmin.html have the same sample codes. I assume we should do the same?

Reply:
SGTM